cppwinrt/component_writers.h

```diff
@@ -289,7 +289,7 @@ catch (...) { return winrt::to_hresult(); }
             bind<write_consume_args>(signature));
     }
 
-    void write_component_static_forwarder(writer& w, MethodDef const& method)
+    inline void write_component_static_forwarder(writer& w, MethodDef const& method)
```

Comment:
This should be static not inline. Look at the rest of the functions in this source file. This helps the compiler issue a warning if the function is unused.